lib/src/test/java/io/ably/lib/transport/HostsTest.java

```diff
@@ -251,21 +233,36 @@ public void hosts_fallback_when_fallback_hosts_and_port_are_defined() throws Abl
      */
     @Test
     public void hosts_fallback_when_host_and_fallback_hosts_and_port_are_defined() throws AblyException {
-        ClientOptions options = new ClientOptions();
+        // Given
         options.tlsPort = 8081;
         options.fallbackHosts = new String[] { "custom-fallback.ably.com" };
+
+        // When
         Hosts hosts = new Hosts("custom.ably.com", Defaults.HOST_REALTIME, options);
 
+        // Then
         assertThat(hosts.getFallback("custom.ably.com"), is("custom-fallback.ably.com"));
     }
 
     @Test(expected = AblyException.class)
     public void hosts_use_default_fallback_hosts_and_tlsport_are_defined() throws AblyException {
-        ClientOptions options = new ClientOptions();
+        // Given
         options.tlsPort = 8081;
         options.fallbackHostsUseDefault = true;
+
+        // When
         Hosts hosts = new Hosts(null, Defaults.HOST_REALTIME, options);
 
+        // Then
         assertThat(hosts.getFallback(Defaults.HOST_REALTIME), nullValue());
```

Comment:
It will never reach this statement right

Reply:
That's right! I will remove this obsolete statement